emspayklarnapaylater/emspayklarnapaylater.php

```diff
@@ -265,9 +265,9 @@ public function execPayment($cart, $locale)
         $description = sprintf($this->l('Your order at')." %s", Configuration::get('PS_SHOP_NAME'));
         $totalInCents = EmspayHelper::getAmountInCents($cart->getOrderTotal(true));
         $currency = EmspayHelper::getPaymentCurrency();
-        $webhookUrl = self::getWebHookUrl();
+        $webhookUrl = EmspayHelper::getWebHookUrl();
```

Comment:
add `returnUrl` in the next line, we use in other payments variable assigned to Return URL, but in this, we take return URL directly while creating an order.